src/hct_mis_api/apps/household/models.py

```diff
@@ -1378,11 +1378,28 @@ def mark_as_duplicate(self, original_individual: Optional["Individual"] = None)
         self.save()
 
     def mark_as_distinct(self) -> None:
-        self.documents.update(status=Document.STATUS_VALID)
-        self.accounts.update(active=True)
-        self.duplicate = False
-        self.duplicate_date = timezone.now()
-        self.save()
+        try:
```

Comment:
To be honest I don't like this solution :)
Maybe we can iterate over each document and update it one by one, then we don;t have to parse exception message etc.

This way also allows to sepreate the accounts logic